### PR TITLE
[sui native bridge] - add update committee node url to validator CLI

### PR DESCRIPTION
**Pull request by patrickkuo**
## Description 

as titled 

## Test plan 

How did you test the new or updated feature?

---

## Release notes

Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.

For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 

- [ ] Protocol: 
- [ ] Nodes (Validators and Full nodes): 
- [ ] Indexer: 
- [ ] JSON-RPC: 
- [ ] GraphQL: 
- [ ] CLI: 
- [ ] Rust SDK:

### CODE REVIEW

**Comment by longbowlu** [comment]
@patrickkuo  did you test this locally?

File: crates/sui/src/validator_commands.rs [hunk 1]
```diff
@@ -182,6 +184,18 @@ pub enum SuiValidatorCommand {
         #[clap(long)]
         validator_address: Option<SuiAddress>,
     },
+    /// Sui native bridge committee member registration
```
Comment (dariorussi) [inline crates/sui/src/validator_commands.rs:187]: this is the wrong comment right?

File: crates/sui/src/validator_commands.rs [hunk 2]
```diff
@@ -182,6 +184,18 @@ pub enum SuiValidatorCommand {
...
+    UpdateBridgeCommitteeNodeUrl {
+        #[clap(long)]
+        bridge_authority_url: String,
```
Comment (dariorussi) [inline crates/sui/src/validator_commands.rs:190]: can you add a comment here so it shows on the help for the command? It is self explanatory but still

File: crates/sui/src/validator_commands.rs [hunk 3]
```diff
@@ -182,6 +184,18 @@ pub enum SuiValidatorCommand {
...
+        /// If true, only print the unsigned transaction and do not execute it.
+        /// This is useful for offline signing.
+        #[clap(name = "print-only", long, default_value = "false")]
+        print_unsigned_transaction_only: bool,
+        /// Must present if `print_unsigned_transaction_only` is true.
```
Comment (dariorussi) [inline crates/sui/src/validator_commands.rs:195]: `Must be present`?

File: crates/sui/src/validator_commands.rs [hunk 4]
```diff
@@ -557,6 +575,83 @@ impl SuiValidatorCommand {
...
+                println!(
+                    "Updating bridge committee node URL for Sui validator: {address}, url: {}",
+                    bridge_authority_url
+                );
+
+                let bridge = bridge_client
+                    .get_mutable_bridge_object_arg_must_succeed()
+                    .await;
+
+                let gas = context
```
Comment (dariorussi) [inline crates/sui/src/validator_commands.rs:622]: would it make sense to check that the gas object has enough balance for the call?
or maybe call `gas_for_owner_budget`? or have the ability to pass a given gas object?

**Review by dariorussi — COMMENTED** [review]
let's wait for @longbowlu to comment again, I'll get back to this later otherwise

File: crates/sui/src/validator_commands.rs [hunk 5]
```diff
@@ -557,6 +575,83 @@ impl SuiValidatorCommand {
                     }
                 }
             }
+            SuiValidatorCommand::UpdateBridgeCommitteeNodeUrl {
+                bridge_authority_url,
+                print_unsigned_transaction_only,
+                validator_address,
+            } => {
+                // Make sure the address is member of the committee
```
[the remark this answers is not in the document]
Comment (longbowlu) [reply crates/sui/src/validator_commands.rs:584]: sorry i forgot to submit this comment yesterday @patrickkuo

**Comment by patrickkuo** [comment]
did a manual local test, can confirm everything works as expected
image


result:

image

**Review by longbowlu — APPROVED** [review]
thanks!

**Review by dariorussi — APPROVED** [review]
thanks

**Pull request merged**